src/ColumnDisplayFormatDialog.h

```diff
@@ -20,7 +20,6 @@ class ColumnDisplayFormatDialog : public QDialog
 public:
     explicit ColumnDisplayFormatDialog(DBBrowserDB& db, const sqlb::ObjectIdentifier& tableName, const QString& colname, QString current_format, QWidget* parent = nullptr);
     ~ColumnDisplayFormatDialog() override;
-
```

Comment:
You should undo this change, so the PR has a clean history when squashed and merged.